lib/src/analysis_options.dart

```diff
@@ -0,0 +1,110 @@
+// Copyright (c) 2017, the Dart project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+const String analysisOptions = '''
+analyzer:
+  strong-mode: true
+
+# Source of linter options:
+# http://dart-lang.github.io/linter/lints/options/options.html
+
+linter:
```

Comment:
Nuke all of these lints. We shouldn't enforce lints on packages – unless we're sure they should be universal

Reply:
I think some of it should be universal, like:
- `annotate_overrides`: being able detect when the implemented interface changes in an unexpected way
- `hash_and_equals`
- `valid_regexps`

The others could be still part of our health score, and therefore present here. Which ones exactly: that is up for debate, but I'd definitely include these:
-  `implementation_imports`: because the package has a known risk factor
- `unawaited_futures`: may be an oversight or an error
- naming and style conventions (like `camel_case_types`), because these help users to build projects with the same universal style